docs/bosh/README.md

```diff
@@ -398,13 +451,16 @@ Now you have the infrastructure ready to deploy a BOSH director.
 terminal multiplexer such as `tmux` or `screen`.
 
    ```
-   bosh-init deploy manifest.yml
+   bosh2 create-env manifest.yml
    ```
 
-1. Target your BOSH environment:
+1. Download [ca_cert.pem](../ca_cert.pem) from this repository.
+
+1. Target your BOSH environment and login:
 
    ```
-   bosh target 10.0.0.6
+   bosh2 alias-env my-bosh-env --environment 10.0.0.6 --ca-cert ca_cert.pem
```

Comment:
suggest 'micro-google' as the env name, this is inline with the director name in the manifest